app/models/course.rb

```diff
@@ -3,6 +3,13 @@ class Course < ActiveRecord::Base
   has_many   :sections, dependent: :destroy
   validates  :number, presence: true, uniqueness: { scope: :department_id }
   default_scope { order(number: :asc) }
+  after_save :respond_course
```

Comment:
can we come up with a more descriptive method name that `respond_course`? That doesn't really have meaning in the context of this class.